.github/workflows/markdownlint.yml

```diff
@@ -0,0 +1,32 @@
+name: Markdownlint
+
+on:
+  push:
+    paths:
+      - "**/*.md"
+      - ".markdownlint.json"
+      - ".github/workflows/markdownlint.yml"
+      - ".github/workflows/markdownlint-problem-matcher.json"
```

Comment:
Are you sure that these paths are required?

Reply:
yeah, it makes sure the CI is triggered when any of the files in the pipeline are touched. Alternately, there could be no filter and it could run on every PR

Reply:
Got it.